packages/server/lib/makeDataContext.ts

```diff
@@ -117,6 +117,18 @@ export function makeDataContext (options: MakeDataContextOptions): DataContext {
       closeActiveProject () {
         return openProject.closeActiveProject()
       },
+      getCurrentProjectSavedState () {
```

Comment:
We should make a note to bring `savedState` into data-context. We should be able to get this data via `data-context`, so the state functionality should be lifted into the ProjectLifecycleManager/config code